install/kubernetes/helm/istio/example-values/values-istio-googleca.yaml

```diff
@@ -23,6 +23,6 @@ nodeagent:
   image: node-agent-k8s
   env:
     CA_PROVIDER: "GoogleCA"
-    CA_ADDR: "istioca.googleapis.com:443"
-    Plugins: "GoogleTokenExchange"
+    CA_ADDR: "meshca.googleapis.com:443"
+    PLUGINS: "GoogleTokenExchange"
```

Comment:
This capitalization brakes the Google CA helm template.
https://github.com/istio/istio/pull/17301 fixes it.